### DISCUSSION TITLE
job is stuck as running, cancel workflow not working

### DISCUSSION THREAD

**Discussion by GurayCetin** [community/community#120060] [Actions] [answered]
### Select Topic Area

Bug

### Body

A job is stuck more than 1.5 hours and looking as running icon even if it's finished. cancel workflow is not working to cancel it.

**Comment by asterix201** [comment, marked as answer]
you can try to force-cancel it 
https://docs.github.com/en/rest/actions/workflow-runs?apiVersion=2022-11-28#force-cancel-a-workflow-run

**Reply by GurayCetin** [reply]
Thank you, it worked!
